packages/template-chakra-storefront/src/pages/login/passwordless-landing.test.js

```diff
@@ -67,35 +65,11 @@ jest.mock('@salesforce/commerce-sdk-react', () => {
     }
 })
 
-// Set up and clean up
-beforeEach(() => {
-    global.server.use(
-        rest.post('*/customers', (req, res, ctx) => {
-            return res(ctx.delay(0), ctx.status(200), ctx.json(mockedRegisteredCustomer))
-        }),
-        rest.get('*/customers/:customerId', (req, res, ctx) => {
-            const {customerId} = req.params
-            if (customerId === 'customerId') {
-                return res(
-                    ctx.delay(0),
-                    ctx.status(200),
-                    ctx.json({
-                        authType: 'guest',
-                        customerId: 'customerid'
-                    })
-                )
-            }
-            return res(ctx.delay(0), ctx.status(200), ctx.json(mockedRegisteredCustomer))
-        })
-    )
-})
```

Comment:
Where did this go or what was it replaced with?

Reply:
From what I see, the tests does not require these mock at all, the tests in this component is checking for redirect logic within the component and does not requires any api mock

Reply:
Gotcha